ml-algorithms/src/main/java/org/opensearch/ml/engine/encryptor/EncryptorImpl.java

```diff
@@ -68,7 +68,8 @@ public String encrypt(String plainText) {
         initMasterKey();
         final AwsCrypto crypto = AwsCrypto.builder().withCommitmentPolicy(CommitmentPolicy.RequireEncryptRequireDecrypt).build();
         byte[] bytes = Base64.getDecoder().decode(masterKey);
-        JceMasterKey jceMasterKey = JceMasterKey.getInstance(new SecretKeySpec(bytes, "AES"), "Custom", "", "AES/GCM/NoPadding");
+        // https://github.com/aws/aws-encryption-sdk-java/issues/1879
+        JceMasterKey jceMasterKey = JceMasterKey.getInstance(new SecretKeySpec(bytes, "AES"), "Custom", "", "AES/GCM/NOPADDING");
```

Comment:
Will this break BWC ?

Reply:
> Will this break BWC ? 

No, the string is uppercased by the method being called. We are just preemptively uppercasing to avoid the wrong uppercase I in Turkish locale.